### DISCUSSION TITLE
Checkbox: add 'indeterminate' prop

### DISCUSSION THREAD

**Discussion by grasdira** [themesberg/flowbite-react#870] [Ideas]
#### Enhancement Suggestion
I think it would be nice to add `indeterminate` (or `isIndeterminate`) prop to Checkbox component.

**Comment by tulup-conner** [comment]
I didn't even realize that's a thing. I'll convert this into an issue under Feature Request. We should definitely support it. I personally prefer prefixing booleans by `is` but in this case the official JS API calls it just `indeterminate` on DOM elements so that's probably best.

**Comment by alebo611** [comment]
Any progress on this? It seems pretty standard nowadays with an indeterminate state, for example a "select all" checkbox that controls checkboxes one level below

**Reply by rluders** [reply]
Check #871 issue. Any PR fixing it would be welcome.

**Comment by tulup-conner** [comment]
Closing in favor of #871 - I should have "converted to issue"
